client/js/middleware/serialization/qbank/deserializers/image_sequence.js

```diff
@@ -8,18 +9,24 @@ function deserializeChoices(choices, correctAnswer, incorrectId) {
   _.forEach(choices, (choice, index) => {
     const answerIndex = correctAnswer.choiceIds.indexOf(choice.id);
     const isCorrect = answerIndex >= 0;
+    const nodes = $('<div>').html(choice.text);
+    const image = $('img', nodes);
+    const label = $('p', nodes);
+
     newChoices[choice.id] = {
       id: choice.id,
       answerId: isCorrect ? correctAnswer.id : incorrectId,
-      text: choice.text,
+      text: image ? image.attr('src') : '',
+      altText: image ? image.attr('alt') : '',
       order: index + 1,
-      answerOrder: index,
+      answerOrder: isCorrect ? answerIndex : null,
```

Comment:
Are we still using answerOrder for anything? If not we could maybe just get rid of it.